Index: Clipboard.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git /usr/local/lib/python3.11/site-packages/label_studio_sdk/client.py  /usr/local/lib/python3.11/site-packages/label_studio_sdk/client.py
--- /usr/local/lib/python3.11/site-packages/label_studio_sdk/client.py
+++ /usr/local/lib/python3.11/site-packages/label_studio_sdk/client.py
@@ -78,6 +78,7 @@
         self.url = url.rstrip('/')
         self.make_request_raise = make_request_raise
         self.session = session or self.get_session()
+        self.cookies = cookies
 
         # set api key or get it using credentials (username and password)
         if api_key is None and credentials is None:
@@ -98,8 +99,6 @@
         if extra_headers:
             self.headers.update(extra_headers)
 
-        # set cookies
-        self.cookies = cookies
 
         # set versions from /version endpoint
         self.versions = versions if versions else self.get_versions()
